Sources/MongoSwift/BSON/Document.swift

```diff
@@ -54,7 +54,7 @@ public struct Document {
     internal var storage: DocumentStorage
 
     /// Returns the number of (key, value) pairs stored at the top level of this `Document`.
-    public fileprivate(set) var count: Int
+    public internal(set) var count: Int
```

Comment:
This change (and the one below) are necessary to use this stuff in `append`, in the future i imagine they'll go back to being private or fileprivate when we work on the memory safety epic.